libvips/foreign/dcrawload.c

```diff
@@ -156,14 +161,26 @@ vips_foreign_load_dcraw_set_metadata(VipsForeignLoadDcRaw *raw,
 
 		g_date_time_unref(dt);
 	}
+#else /*!GLIB_CHECK_VERSION(2, 62, 0)*/
+	GTimeVal val = { raw->raw_processor->other.timestamp, 0 };
+
+	char *str = g_time_val_to_iso8601(&val);
+	if (str) {
+		vips_image_set_string(image, "raw-timestamp", str);
+
+		g_free(str);
+	}
+#endif /*GLIB_CHECK_VERSION(2, 62, 0)*/
 
+#if LIBRAW_COMPILE_CHECK_VERSION_NOTLESS(0, 17)
 	if (raw->raw_processor->idata.xmpdata)
 		vips_image_set_blob_copy(image, VIPS_META_XMP_NAME,
 			raw->raw_processor->idata.xmpdata,
 			raw->raw_processor->idata.xmplen);
```

Comment:
Introduced in commit https://github.com/LibRaw/LibRaw/commit/af4afda1ee093162fcd9f9e9c116033bf42ef0f1.